integration_tests/tests/people/timeline.cy.ts

```diff
@@ -13,27 +13,29 @@ context('Application timeline', () => {
     // Given I am logged in
     cy.signIn()
   })
+  ;([true, false] as const).forEach(isOfflineApplication => {
+    it(`shows a timeline for a CRN if is offline application ${isOfflineApplication}`, () => {
```

Comment:
I think `shows the timeline for a CRN if isOfflineApplication is: ${isOfflineApplication}` would be clearer